src/main/ffmpeg-optimizations-fixed.ts

```
@@ -0,0 +1,227 @@
+    'copy': {
+      fast: ['-c', 'copy'],
+      balanced: ['-c', 'copy'],
+      quality: ['-c', 'copy']
+    }
+  };
+  
+  return presets[codec as keyof typeof presets]?.[quality] || ['-c', 'copy'];
+}
+
+// Optimization 7: Smart quality detection
+export function detectOptimalQuality(_inputFile: string, streams: any[]): 'fast' | 'balanced' | 'quality' {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)